### DISCUSSION TITLE
Поле типа "Селект", свойства, как включить индикатор "стрелка вниз" и как включить одиночный клик левой кнопки мышки?

### DISCUSSION THREAD

**Discussion by IgorVolod** [totumonline/totum-ru-issues-and-discussions#620] [Как это сделать?] [answered]
Field типа "Селект":
1. Как включить индикатор, обозначающий "выпадающий список" справа в виде черной стрелки вниз?
2. Как включить режим отображения выпадающего списка по одиночному клику левой кнопки мышки?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Код форматирования поля:

```
f1=: setFormat(editbutton: true; icon: "caret-down")
```

Открытие будет по одинарному клику на иконку редактирования справа

**Comment by IgorVolod** [comment]
Спасибо! Если подскажете, как еще сами кнопки раскрасить, было бы пределом мечтаний.)
![Безымянный](https://github.com/user-attachments/assets/887775cc-de22-4f69-baa6-f68ae4268ff4)
Стало вот так... а сделать, как у простых людей, справа стрелка, без стрелки слева, можно?

**Reply by ttm-alexeypolunin** [reply]
В стандартном виде у кнопки красится бекграунд поля. Если в хедере/футере вы можете сделать Секцию и отключить полю кнопки обводку.

https://ru.docs.totum.online/interface#sx-section-assignment

**Reply by ttm-alexeypolunin** [reply]
Тогда код форматирования

```
f2=: setFormat(background: "tomato"; color: white)
```

Будет закрашивать тело кнопки

**Reply by ttm-alexeypolunin** [reply]
Эти кнопки крясятся вместе с цветом текста.

Уберите стрелочку она там лишняя.

**Reply by IgorVolod** [reply]
Убрал стрелку слева... стало так:
![Безымянный2](https://github.com/user-attachments/assets/ec1bf056-9e9c-4579-ae2c-c5b46cc0dd11)
Клик по иконке: 
![Безымянный3](https://github.com/user-attachments/assets/4ab0bb2b-1dc9-40ef-b16f-977185df0ab1)
Произошла подмена : иконка стала стрелкой вниз, шрифт изменился, поле стало серого цвета.?  Сверху открылось меню, которое в этом месте лишнее.
